unit/models/transaction.py

```diff
@@ -114,12 +113,12 @@ def from_json_api(_id, _type, attributes, relationships):
 
 class BookTransactionDTO(BaseTransactionDTO):
     def __init__(self, id: str, created_at: datetime, direction: str, amount: int, balance: int,
-                 summary: str, counterparty: Counterparty, tags: Optional[Dict[str, str]],
-                 relationships: Optional[Dict[str, Relationship]]):
+                 summary: str, description: str, addenda: Optional[str], counterparty: Counterparty,
```

Comment:
for sure description & addenda should be here? https://docs.unit.co/resources/#attributes-25

Reply:
when you are right you are right